applications/party/servicedef/services_view.xml

```diff
@@ -74,14 +74,15 @@ under the License.
     <service name="performFindParty" engine="java"
         location="org.apache.ofbiz.party.party.PartyServices" invoke="performFindParty">
         <description>General Party Find Service, duplicated for screen widget purpose, Used in the new findparty page in the Party Manager, etc</description>
-        <attribute name="roleTypeId" type="String" mode="IN" optional="true"/> <!-- can be null or ANY to include any -->
         <attribute name="noConditionFind" type="String" mode="IN" optional="true" default-value="N"/>
         <attribute name="extInfo" type="String" mode="IN" optional="true"/>
         <attribute name="extCond" type="org.apache.ofbiz.entity.condition.EntityCondition" mode="IN" optional="true">
             <description>EntityCondition that can be send to this service to manage complex search case</description>
         </attribute>
         <attribute name="partyId" type="String" mode="IN" optional="true"/> <!-- does a LIKE compare on this, can do partial, case insensitive, etc -->
         <attribute name="partyTypeId" type="String" mode="IN" optional="true"/>
+        <attribute name="roleTypeId" type="String" mode="IN" optional="true"/>
```

Comment:
I think this is unwanted change. Could you please revert?

Reply:
I agree I see no reason why
        <attribute name="roleTypeId" type="String" mode="IN" optional="true"/> <!-- can be null or ANY to include any -->
has been moved. Not a big deal but we lose the comment